#!/bin/sh -e
## DP: Upgrade paths to match the package's routes

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1;;
esac

exit 0
=== modified file 'customize-cd-gui'
--- ../uck/uck-gui	2006-12-22 20:26:32.000000000 +0100
+++ ./uck-gui	2007-02-15 21:09:31.000000000 +0100
@@ -115,7 +115,7 @@
 # EXECUTION STARTS HERE #
 #########################

-. libraries/gui.sh
+. /usr/lib/uck/gui.sh

 function gui_failure()
 {
@@ -224,9 +224,10 @@
 echo "Preparing build environment..."
 BUILD_DIR=`mktemp -d`

-SCRIPT_DIR=`dirname "$0"`
-cp -a "$SCRIPT_DIR"/customization-profiles/localized_cd/* "$BUILD_DIR"/
-cp -a libraries/gui.sh "$BUILD_DIR"/
+CUST_SCRIPT_DIR=/usr/lib/uck/scripts
+SCRIPT_DIR=/usr/bin
+cp -a "$CUST_SCRIPT_DIR"/customization-profiles/localized_cd/* "$BUILD_DIR"/
+cp -a /usr/lib/uck/gui.sh "$BUILD_DIR"/

 echo "$CD_LANGS" >"$BUILD_DIR"/language_packs
 echo "$BOOT_CD_LANG" >"$BUILD_DIR"/livecd_locale

=== modified file 'remaster-live-cd'
--- ../uck/remaster-live-cd	2007-01-02 16:37:02.000000000 +0100
+++ ./remaster-live-cd	2007-02-15 21:10:08.000000000 +0100
@@ -41,7 +41,7 @@

 export LC_ALL=C

-. libraries/remaster-live-cd.sh
+. /usr/lib/uck/remaster-live-cd.sh

 trap unmount_all EXIT
 trap unmount_all SIGINT